### DISCUSSION TITLE
How to build Agent

### DISCUSSION THREAD

**Discussion by vishalkadam18** [apache/skywalking#12181] [Q&A]
Hi Team,

I am trying to build agent with some changes but that unable to build can you guide me??

Regards,
Vishal.

**Comment by Superskyyy** [comment]
Hello there! The official docs already included the proper way to build the agent. What you did privately to modify the agent is unfortunately out of our capacity to help with.

**Comment by wu-sheng** [comment]
No matter you like or not, we can't afford to debug for everyone.
As @Superskyyy applied, your changes mean your responsibilities. And you just said can't, who knows why? Your asking doesn't make sense.

Docs are there, try to read by yourself. https://skywalking.apache.org/docs/skywalking-java/latest/en/contribution/compiling/
